cmd/manager/main.go

```diff
@@ -48,7 +52,18 @@ func main() {
 
 	// Create a new Cmd to provide shared dependencies and start components
 	log.Info("setting up manager")
-	mgr, err := manager.New(cfg, manager.Options{})
+	// by default manager sets up DelegatingClient which reads from cache and not directly from API server
```

Comment:
shouldn't this be done in manager?   this calls manager to call manager with configuration options.
should we always have this type of Manager / Client... if not could we have 2 types of News... `New` and `NewCachless`

Reply:
Until we re-write controller-runtime for ourselves, yes. We don't own manager, manager is part of controller-runtime.

In the meantime, I've just asked for the implications of not using the opinionated controller-runtime stack on performance.

Also just reveals a fun Go-ism of controller-runtime/manager's Options struct, and us having not enough separation of concerns between CLI code and _code that does things_™.